### PR TITLE
fix: getFee- returned 0 in replyState

### PR DESCRIPTION
**Pull request by gcranju**
### Commit Message

```bash
fix: getFee- returned 0 in replyState
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: contracts/evm/contracts/xcall/CallService.sol [hunk 1]
```diff
@@ -635,8 +635,8 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
         string[] memory _sources
     ) external view override returns (uint256) {
         uint256 fee = protocolFee;
-        if (isReply(_net, _sources) && !_rollback) {
-            return fee;
+        if (isReply(_net, _sources)) {
```
Comment (AntonAndell) [inline contracts/evm/contracts/xcall/CallService.sol:638]: Should it not still be && !_rollback ?

Comment (gcranju) [reply contracts/evm/contracts/xcall/CallService.sol:638]: Ah, I guess it should be, I probably checked with java as there is not check for rollback, and changed I guess, so need to change in both right?

Comment (AntonAndell) [reply contracts/evm/contracts/xcall/CallService.sol:638]: Yes, It seems to have been missed in Java

**Review by AntonAndell — APPROVED** [review]
Looks good

**Pull request merged**